contents/dataframesmeta_chain.md

```diff
@@ -0,0 +1,67 @@
+## Piping Operations {#sec:dataframesmeta_chain}
+
+R users are familiar with the pipe operator `%>%`.
+It allows you to **chain operations together**.
+That means that the output of an operation will be used as input in the next operation and so on.
+
+This can be accomplished with the `@chain` macro.
+The way it works is quite simple.
+First, we start with `@chain` followed by the `DataFrame` and a `begin` statement.
+Every operation inside the `begin ... end` statement will be used as input for the next operation,
+therefore chaining operations together.
+
+Here's a simple example with a `groupby` followed by a `@combine`:
+
+```jl
+sco("""
+@chain leftjoined begin
+    groupby(:name)
+    @combine :mean_grade_2020 = mean(:grade_2020)
+end
+"""; process=without_caption_label
+)
+```
+
+> **_NOTE:_**
+> `@chain` will replace the first positional argument while chaining operations.
+> This is not a problem in `DataFrames.jl` and DFM,
+> since the `DataFrame` is always the first positional argument.
+
+We can also nest as many as `begin ... end` statements we desired inside the operations:
+
+```jl
+sco("""
+# second @chain # hide
```

Comment:
Sorry for not having fixed this yet :sweat: (https://github.com/JuliaBooks/Books.jl/issues/177)